configure.ac

```diff
@@ -829,10 +829,10 @@ VIPS_LIBS="$VIPS_LIBS $NIFTI_LIBS"
 AC_ARG_WITH([libjxl],
   AS_HELP_STRING([--without-libjxl], [build without libjxl (default: test)]),
   [with_libjxl=$withval],
-  [with_libjxl=$gmodule_with_flag])
+  [with_libjxl=test])
```

Comment:
Nice, this means we can get rid of the `gmodule_with_flag` variable if we do this for all libraries.